services/src/northbound/src/main/java/org/openkilda/northbound/service/impl/FlowServiceImpl.java

```diff
@@ -317,13 +323,8 @@ public FlowPayload updateFlow(final FlowPayload flow) {
     public FlowIdStatusPayload statusFlow(final String id) {
         final String correlationId = RequestCorrelationId.getId();
         LOGGER.debug("Flow status: {}={}", CORRELATION_ID, correlationId);
-        FlowStatusRequest data = new FlowStatusRequest(new FlowIdStatusPayload(id, null));
-        CommandMessage request = new CommandMessage(data, System.currentTimeMillis(), correlationId, Destination.WFM);
-        messageConsumer.clear();
-        messageProducer.send(topic, request);
-        Message message = (Message) messageConsumer.poll(correlationId);
-        FlowStatusResponse response = (FlowStatusResponse) validateInfoMessage(request, message, correlationId);
-        return response.getPayload();
+        BidirectionalFlow flow = getBidirectionalFlow(id, correlationId);
+        return FlowPayloadToFlowConverter.buildFlowIdStatusPayload(flow);
```

Comment:
mapstruct

Reply:
This `FlowIdStatusPayload` requires the `message` package.
Should I forcibly apply mapstruct and add something like [this](https://github.com/telstra/open-kilda/blob/e04f09e2436c472576be58996f3596666ae67a3d/services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowPayload.java#L33) comment?